dotcom-rendering/src/lib/useCommentCount.ts

```diff
@@ -4,32 +4,35 @@ import { useApi } from './useApi';
 
 type CommentCounts = Record<string, number>;
 
-const getInitialSetFromDOMAttribute = (attribute: string) =>
-	[...document.querySelectorAll(`[${attribute}]`)]
-		.map((element) => element.getAttribute(attribute))
-		.filter(isNonNullable);
+const DATA_ATTRIBUTE = 'data-discussion-id';
```

Comment:
I've always wondered what this pattern is for? My worry is that it's previously allowed someone to export it, and then couple their modules (I can't remember where I've seen this) and why not just use the string?

Reply:
What about [more coupling](https://github.com/guardian/dotcom-rendering/pull/8628)? This actually makes it very explicit that there is a dependency between these two values.

Planning to revisit how to improve this overall pattern in #8616